x/ccv/child/keeper/relay.go

```diff
@@ -32,6 +32,8 @@ func (k Keeper) OnRecvPacket(ctx sdk.Context, packet channeltypes.Packet, newCha
 		// Set CCV channel status to Validating and set parent channel
 		k.SetChannelStatus(ctx, packet.DestinationChannel, ccv.VALIDATING)
 		k.SetParentChannel(ctx, packet.DestinationChannel)
+		// Send pending slash requests in states
+		k.SendPendingSlashRequest(ctx)
```

Comment:
why are we only sending slashing packets on recv packet?

Reply:
Is the logic from the [spec]( https://github.com/cosmos/ibc/blob/marius/ccv/spec/app/ics-028-cross-chain-validation/technical_specification.md#ccv-ccf-rcvvsc1), i.e., all the **_pending_** slash packets are sent once the CCV channel is established. This happens on receiving the first VSC packet.